docfx.json

```diff
@@ -645,6 +646,13 @@
             "enable_powershell_try_it": {
                 "azurermps-*/**/*.md": true,
                 "azps-*/**/*.md": true
+            },
+            "searchScope": {
+               "az*-*/Az*.DataLakeAnalytics/*.md": ["Azure Data Lake Analytics","Azure"],
```

Comment:
Is there a particular reason why "Azure" should be part of these search scopes? It seems like typing 'Azure' in the LHS search bar would result in a lot of spam.

Reply:
That's the default for the repo...  just going with the flow:
https://github.com/MicrosoftDocs/azure-docs-powershell/blob/master/docfx.json#L654
```
        "globalMetadata": {
            "apiPlatform": "powershell",
            "brand": "azure",
            "searchScope": [
                "Azure"
            ],
```

I think it would be fine to keep in the Azure scope (so that Azure searches include Az PowerShell stuff), but to make it secondary. I think we should make a new scope for this repo that is "Azure PowerShell" so that customers can search within just this scope. I'm happy to help with that effort. It's a oneline change I think. The search index takes a day or two to build, so it will be in partial limbo until the search crawler does its thing. 

Thanks, Jason

Reply:
@sptramer Trying that so you can see how it look. The search results won't work on the review site, but the TOC filter will simulate the appearance.

Reply:
@sptramer  How's this:
![image](https://user-images.githubusercontent.com/5067358/66440083-a4508100-e9e6-11e9-9c81-165d42e21d5a.png)

Reply:
LGTM, I can merge if you're ready.

Reply:
I think so #sign-off 
We can test the live site in a day or two to make sure the scope is working as expected.